<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="hostEIDLabel">
            <property name="text">
             <string>Host EID</string>
            </property>
            <property name="buddy">
             <cstring>hostEIDLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="hostEIDLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="portLabel">
            <property name="text">
             <string>Port</string>
            </property>
            <property name="buddy">
             <cstring>portSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="portSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="nicknameLabel">
            <property name="text">
             <string>Nickname</string>
            </property>
            <property name="buddy">
             <cstring>nicknameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="nicknameLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="firstNameLabel">
            <property name="text">
             <string>First name</string>
            </property>
            <property name="buddy">
             <cstring>firstNameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="firstNameLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lastNameLabel">
            <property name="text">
             <string>Last name</string>
            </property>
            <property name="buddy">
             <cstring>lastNameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lastNameLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="cityLabel">
            <property name="text">
             <string>City</string>
            </property>
            <property name="buddy">
             <cstring>cityLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="cityLineEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="regionLabel">
            <property name="text">
             <string>Region</string>
            </property>
            <property name="buddy">
             <cstring>regionLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="regionLineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="countryLabel">
            <property name="text">
             <string>Country</string>
            </property>
            <property name="buddy">
             <cstring>countryLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="countryLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Routing servers:</string>
            </property>
            <property name="buddy">
             <cstring>plainTextEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Generate host ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>&amp;Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>&amp;Quit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGenerate_new_host_ID"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionGenerate_new_host_ID">
   <property name="text">
    <string>Generate new host ID</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>nicknameLineEdit</tabstop>
  <tabstop>firstNameLineEdit</tabstop>
  <tabstop>lastNameLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>regionLineEdit</tabstop>
  <tabstop>countryLineEdit</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>hostEIDLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
